diffusion/models/models.py

```diff
@@ -311,6 +313,8 @@ def stable_diffusion_xl(
             # Adapt the unet config to account for differing number of latent channels if necessary
             unet_config['in_channels'] = vae.config['latent_channels']
             unet_config['out_channels'] = vae.config['latent_channels']
+        if model_name == 'sdxl-e5':  # e5 + clip embedding dims + micro conditioning
+            unet_config]['cross_attention_dim'] = 2304
```

Comment:
```suggestion
            unet_config['cross_attention_dim'] = 2304
```

fix missing brackets